src/reference/liveloading/events.json

```diff
@@ -14,6 +14,7 @@
     "user:{id}:followed": "Sent when a user follows or unfollows a channel. The body will contain an attribute `following` - boolean set to true `true` if the user just followed the channel, `false` if they unfollowed, and user - the [User](/rest.html#User) who just followed the channel.",
     "user:{id}:notify": "Sent when a new user [Notification](/rest.html#Notification) is created. Notifications of the same type will always follow the same structure.",
     "user:{id}:subscribed": "Sent when the user subscribes to a channel. The body will contain the channel ID that they just subscribed to.",
+    "user:{id}:resubscribed": "Sent when an automatic resubscription to a channel happens. The body will contain the [Channel](/rest.html#Channel) id of the channel the user just resubscribed to. Additionally there is a `since` and `until` property. The are the dates for when the user first subscribed (Counts from the start of recurring billing period) and when the subscription expires. Finally there's also a `totalMonths` property, which is an integer of how many months the user has been subscribed since the beginning of time.",
```

Comment:
```
Sent when an automatic resubscription to a channel happens. The body will contain the [Channel](/rest.html#Channel) id of the channel the user just resubscribed to. Additionally there are `since` and `until` properties. These are the dates for when the user first subscribed (from the start of recurring billing period) and when the subscription expires. Finally there's a `totalMonths` property, which is an integer indicating how many months the user has been subscribed since the beginning of time.
```

fixed it :P